client/views/chan.tpl

```diff
@@ -1,7 +1,7 @@
 {{#each channels}}
 <div data-id="{{id}}" data-target="#chan-{{id}}" data-title="{{name}}" class="chan {{type}} chan-{{slugify name}}">
 	<span class="badge{{#if highlight}} highlight{{/if}}">{{#if unread}}{{roundBadgeNumber unread}}{{/if}}</span>
-	<button class="close" aria-label="Close"></button>
+	{{#if needsCloseButton}}<button class="close" aria-label="Close"></button>{{/if}}
```

Comment:
Test that type is lobby instead, that way you don't have to add `needsCloseButton` everywhere on the server-side (it's a client/front-end implementation detail, so the server doesn't need to know about it).

Reply:
Yeah, that's the first way I tried doing it. When I looked it up, however, I read that handlebars templates are supposed to be logicless and therefore there are no equality operators, so I couldn't find a way to do that.

If you could point me in the right direction that would be great 😃

Reply:
I think that something like [this](https://github.com/thelounge/lounge/blob/master/client/views/chat.tpl#L5) (except not `"channel"`) would work.

Reply:
Ah, alright, that makes sense. Thanks!

I should be able to wrap this up on Sunday.